liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -1,6 +1,5 @@
 /*
- * JBoss, Home of Professional Open Source
- * Copyright 2012, 2013, Red Hat Middleware LLC, and individual contributors
+ * Copyright 2012, 2018, IBM Corporation, Red Hat Middleware LLC, and individual contributors
  * by the @authors tag. See the copyright.txt in the distribution for a
```

Comment:
This refers to the authors tag, but those are removed below; not sure I'd do that; and I'm not sure adding IBM and Red Hat in the same line is the preferred format; I thought it will be multiple lines, but whatever works for you is fine for me in this case; I'd just ask you to re-check that.